### PR TITLE
Fixing LGTM warns + adding dotnet-format

### PR DESCRIPTION
**Pull request by eddynaka**

### CODE REVIEW

File: src/BinSkim.Rules/PERules/WindowsBinaryAndPdbSkimmerBase.cs [hunk 1]
```diff
@@ -80,10 +80,6 @@ public sealed override AnalysisApplicability CanAnalyze(BinaryAnalyzerContext co
                 // Could not locate the PDB for '{0'}. Probing details:{1}
                 : RuleResources.PdbLoadFailed;
 
-            FailureLevel failureLevel = pdbLoadSucceeded
-                ? FailureLevel.Note
-                : FailureLevel.Warning;
-
             context.Logger.LogConfigurationNotification(
```
Comment (eddynaka) [inline src/BinSkim.Rules/PERules/WindowsBinaryAndPdbSkimmerBase.cs:83]: If we check, failurelevel is hardcoded as warning. Should we use the variable that I just removed?

**Comment by michaelcfanning** [comment]
>                     FailureLevel.Warning,

yes, replace this with 'failureLevel' good bug.

---
Refers to: src/BinSkim.Rules/PERules/WindowsBinaryAndPdbSkimmerBase.cs:88 in 88e4200. [](commit_id = 88e4200c2f21ffbf92dd2fcf7cf332f3f03d24bc, deletion_comment = False)

**Review by michaelcfanning — CHANGES_REQUESTED** [review]
:clock1:

**Comment by eddynaka** [comment]
>                     FailureLevel.Warning,

done

---
In reply to: [735307674](https://github.com/microsoft/binskim/pull/312#issuecomment-735307674) [](ancestors = 735307674)

---
Refers to: src/BinSkim.Rules/PERules/WindowsBinaryAndPdbSkimmerBase.cs:88 in 88e4200. [](commit_id = 88e4200c2f21ffbf92dd2fcf7cf332f3f03d24bc, deletion_comment = False)

File: src/BinaryParsers/PEBinary/ProgramDatabase/MSDiaComWrapper.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ internal class MsdiaComWrapper
         private static void CoCreateFromMsdia(Guid clsidOfServer, Guid riid, out IntPtr pvObject)
         {
             IntPtr pClassFactory = IntPtr.Zero;
-            int hr = DllGetClassObject(clsidOfServer, new Guid("00000001-0000-0000-C000-000000000046"), out pClassFactory);
+            _ = DllGetClassObject(clsidOfServer, new Guid("00000001-0000-0000-C000-000000000046"), out pClassFactory);
```
Comment (michaelcfanning) [inline src/BinaryParsers/PEBinary/ProgramDatabase/MSDiaComWrapper.cs:29]: >DllGetClassObject [](start = 16, length = 17)

This code should really check the HRESULT and throw if it isn't S_OK.

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Pull request merged**